add text to pde_constrained.md (#324)
SciML/DiffEqFlux.jl: docs/src/examples/pde_constrained.md

t        = t0:dt:tMax;
```

> `x` (to `Lx`) spans the specified 1D space
> 
> `dx` = distance between two points
> 
> `Nx` = total size of space
> 
> `u0` = initial condition
> 
> `p` = true solution
> 
> `xtrs` = convenient grouping of `dx` and `Nx` into Array
> 
> `dt` = time distane between two points
> 
> `t` (`t0` to `tMax`) spans the specified time frame
> 
> `tspan` = span of `t`

Reviewer: ```suggestion
In plain terms, the quantities that were defined are:

- `x` (to `Lx`) spans the specified 1D space
- `dx` = distance between two points
- `Nx` = total size of space
- `u0` = initial condition
- `p` = true solution
- `xtrs` = convenient grouping of `dx` and `Nx` into Array
- `dt` = time distane between two points
- `t` (`t0` to `tMax`) spans the specified time frame
- `tspan` = span of `t`
```

function d2dx(u,dx)
    """
    2nd order Central difference for 2nd degree derivative
    """
    return [[zero(eltype(u))]; (u[3:end] - 2.0.*u[2:end-1] + u[1:end-2]) ./ (dx^2); [zero(eltype(u))]]
end
```

### Burgers' Differential Equation

Next, we setup our desired set of equations in order to define our problem.

```
## ODE description of the Physics:
function burgers(u,p,t)
    # Model parameters
    a0, a1 = p
    dx,Nx = xtrs #[1.0,3.0,0.125,100]
    return 2.0*a0 .* u +  a1 .* d2dx(u, dx)
end
```

### Solve and Plot Ground Truth

We then solve and plot our partial differential equation. This is the true solution which we
will compare to further on.

```
# Testing Solver on linear PDE
prob = ODEProblem(burgers,u0,tspan,p)
sol = solve(prob,Tsit5(), dt=dt,saveat=t);

plot(x, sol.u[1], lw=3, label="t0", size=(800,500))
plot!(x, sol.u[end],lw=3, ls=:dash, label="tMax")
```

> ### Using Neural Network for Predictions

Reviewer: Let's not call it a neural network. It's just a forward model for the PDE, or a prediction function.

Author: That's fair. Should I modify the 1-layer neural network comment as well?

Reviewer: yes

### Plotting Prediction vs Ground Truth

The scatter points plotted here are the ground truth obtained from the actual solution we
solved for above. The solid line represents our prediction. The goal is for both to overlap

> almost perfectly when the NN finishes its training and the loss is close to 0.

Reviewer: ```suggestion
almost perfectly when the PDE finishes its training and the loss is close to 0.
```

### Train

> The network is trained using `sciml_train` via backpropagation. The specifics of our
> 1-layer NN is stored in `res` and `res.minimizer` returns the parameters that
> minimizes the cost function.

Reviewer: ```suggestion
The parameters are trained using `sciml_train` and adjoint sensitivities. The resulting
best parameters are stored in `res` and `res.minimizer` returns the parameters that
minimizes the cost function.
```
